### PR TITLE
Use Snapshot GraphQL API

### PR DESCRIPTION
**Pull request by glmaljkovich**
**Summary**

I refactored `src/scrape.ts` to use Snapshot's GraphQL API. The calls are paginated and the endpoint is configurable through env variables. The Snapshot spaces are also configurable. I also removed the data files from git and made sure the script creates them if they are not already there. There's also a Makefile (intended to replace the bash script) to run all the commands and ensure dependencies are met.

What I didn't get into this PR yet is configuring the IPFS source and target urls.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+	yarn scrape-snapshot
+
+ipfs-sync: data
+	@echo "Running ipfs-sync..."
+	yarn sync-proposals
+	yarn sync-proposals-relayers
+	yarn sync-votes
+	yarn sync-votes-relayers
+
+scrape:
```
Comment (abarmat) [inline Makefile:22]: When I run `make scrape` I get:

```
scrape-snapshot
make: scrape-snapshot: No such file or directory
make: *** [scrape] Error 1
```

Comment (abarmat) [reply Makefile:22]: If I replace it with `scrape: scrape-snapshot` it works

Comment (glmaljkovich) [reply Makefile:22]: Ah, good catch. Seems I'm a bit rusty with Make :grin:

**Comment by abarmat** [comment]
> What I didn't get into this PR yet is configuring the IPFS source and target urls.

We can work on that on a different PR

**Review by abarmat — APPROVED** [review]

**Pull request merged**